tests/ignite/contrib/metrics/regression/test_fractional_bias.py

```diff
@@ -0,0 +1,46 @@
+from ignite.exceptions import NotComputableError
+from ignite.contrib.metrics.regression import FractionalBias
+import torch
+import numpy as np
+import pytest
+
+
+def test_zero_div():
+    m = FractionalBias()
+    with pytest.raises(NotComputableError):
+        m.compute()
+
+
+def test_fractional_bias():
+    a = np.random.randn(4)
+    b = np.random.randn(4)
+    c = np.random.randn(4)
+    d = np.random.randn(4)
+    ground_truth = np.random.randn(4)
+
+    m = FractionalBias()
+    m.reset()
```

Comment:
No need to do `reset` here, it would be called automatically by `__init__`. As in https://github.com/pytorch/ignite/blob/master/ignite/metrics/metric.py#L23

Same for other metrics.

Reply:
Incorporated this comment.